### PR TITLE
[16.0][FIX] web_company_color: unaligned button view

### PR DESCRIPTION
**Pull request by dsolanki-initos**
Fixed unaligned button view of `web_company_color`

### CODE REVIEW

**Review by hbrunn — APPROVED** [review]

File: web_company_color/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Web Company Color",
     "category": "web",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.0.1",
```
Comment (pedrobaeza) [inline web_company_color/__manifest__.py]: Don't change manually the version.

**Review by pedrobaeza — COMMENTED** [review]
Commit message shouldn't have the Odoo version.

File: web_company_color/__manifest__.py (hunk 1 above)
Comment (dsolanki-initos) [reply web_company_color/__manifest__.py]: OKay

**Comment by dsolanki-initos** [comment]
Hi @pedrobaeza, Made the changes as you said, Kindly check it.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**